app/webapp/src/main/java/uk/ac/ebi/fg/annotare2/web/gwt/common/client/view/DataFilesUploadViewImpl.java

```diff
@@ -418,17 +418,25 @@ public void onFileAdded(ResumableUploader uploader, ResumableFile file) {
         public void onFilesAdded(ResumableUploader uploader, JsArray<ResumableFile> files) {
             boolean shouldUpload = true;
             boolean fileBlocked = false;
+            boolean invalidFileName = false;
             StringBuilder sb = new StringBuilder();
```

Comment:
Not related to this PR but I think it is a good opportunity to rename this string builder to something useful e.g. `duplicateFiles`